Dockerfile

```diff
@@ -87,12 +87,12 @@ ARG WITH_SQLFLOW_MODELS="ON"
 # NOTE: The sqlflow_models works well on the specific Tensorflow version,
 #       we can skip installing sqlflow_models if using the older Tensorflow.
 RUN if [ "${WITH_SQLFLOW_MODELS:-ON}" = "ON" ]; then \
-  git clone https://github.com/sql-machine-learning/models.git && \
-  cd models && \
-  git checkout 58f4c137129e2bc749320bafcc8fddb7c737fed9 && \
+  git clone https://github.com/typhoonzero/models-1.git && \
```

Comment:
Why installing from a fork? Shall we merge the change to `models` repo first?

Reply:
This is for testing the code, will move to develop